src/source/Ice/Network.c

```diff
@@ -331,6 +331,38 @@ STATUS socketWrite(INT32 sockfd, const void* pBuffer, SIZE_T length)
     return retStatus;
 }
 
+BOOL isIpAddr(PCHAR hostname)
```

Comment:
A simple test for this function as well.

Reply:
Done